framework-docs/modules/ROOT/pages/testing/mockmvc/assertj/assertions.adoc

```diff
@@ -47,3 +47,16 @@ include-code::./FamilyControllerTests[tag=assert-file,indent=0]
 
 If you prefer to use another library, you can provide an implementation of
 {spring-framework-api}/test/json/JsonComparator.html[`JsonComparator`].
+
+Alternatively, you can use AssertJ https://github.com/assertj/assertj/blob/3.x/assertj-core/src/main/java/org/assertj/core/api/AssertFactory.java[AssertFactory].
+For example, if you want to use https://github.com/lukas-krecan/JsonUnit[JsonUnit] you can
+do this
+
+```
+import static net.javacrumbs.jsonunit.assertj.JsonAssertions.jsonUnitJson;
```

Comment:
This isn't markdown. We don't compile against JSonUnit so we can't add the example.